### PR TITLE
PT-1955: Collect CloudWatch metrics for all RDS instances automatically

### PR DESCRIPTION
**Pull request by CVirus**
- Fix wrong renamings

### CODE REVIEW

**Comment by bghellofresh** [comment]
+1

**Comment by asaif** [comment]
+1

File: ci/pipeline.yml [hunk 1]
```diff
@@ -545,6 +546,7 @@ jobs:
             chart: hf-charts/rds-exporter
             values:
             - source-code/ahoy/rds-exporter/values-live.yaml
+            - source-code/ahoy/rds-exporter/secrets-live.yaml
```
Comment (aleksandrzhiliaev) [inline ci/pipeline.yml]: please use vault injection

File: ci/pipeline.yml [hunk 2]
```diff
@@ -460,6 +460,7 @@ jobs:
             chart: hf-charts/rds-exporter
             values:
             - source-code/ahoy/rds-exporter/values-staging.yaml
+            - source-code/ahoy/rds-exporter/secrets-staging.yaml
```
Comment (aleksandrzhiliaev) [inline ci/pipeline.yml]: please use vault injection

Comment (CVirus) [reply ci/pipeline.yml]: done

**Comment by rafaeljesus** [comment]
:shipit:

**Pull request merged**